templates/gallery/gallery.amp.json

```diff
@@ -16,7 +16,68 @@
+          }
+        },
+        {
+          "link": {
+            "nav-item-classes": ["gallery-sidebar-nav-item"],
+            "url": "./gallery-inner.amp.html",
+            "text": "<div class='gallery-sidebar-nav-item-text'>Cage<br/>Ellen Doe</div><amp-img class='gallery-sidebar-nav-item-image' width='1' height='1' layout='responsive' src='../../img/gallery/court.jpg' srcset='../../img/gallery/court.jpg 1x, ../../img/gallery/court@2x.jpg 2x'></amp-img><div class='gallery-sidebar-nav-item-icon'><svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 100 100'><polyline fill='none' stroke='currentColor' points='27 5 72.167 50.167 27.335 95'/></svg></div>"
+          }
+        },
+        {
+          "link": {
+            "nav-item-classes": ["gallery-sidebar-nav-item"],
+            "url": "./gallery-inner.amp.html",
+            "text": "<div class='gallery-sidebar-nav-item-text'>Living Tree<br/>David Beck</div><amp-img class='gallery-sidebar-nav-item-image' width='1' height='1' layout='responsive' src='../../img/gallery/orange.jpg' srcset='../../img/gallery/orange.jpg 1x, ../../img/gallery/orange@2x.jpg 2x'></amp-img><div class='gallery-sidebar-nav-item-icon'><svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 100 100'><polyline fill='none' stroke='currentColor' points='27 5 72.167 50.167 27.335 95'/></svg></div>"
+          }
+        },
+        {
+          "link": {
+            "nav-item-classes": ["gallery-sidebar-nav-item"],
+            "url": "./gallery-inner.amp.html",
+            "text": "<div class='gallery-sidebar-nav-item-text'>Short Circuit<br/>Anna Moss</div><amp-img class='gallery-sidebar-nav-item-image' width='1' height='1' layout='responsive' src='../../img/gallery/lamp.jpg' srcset='../../img/gallery/lamp.jpg 1x, ../../img/gallery/lamp@2x.jpg 2x'></amp-img><div class='gallery-sidebar-nav-item-icon'><svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 100 100'><polyline fill='none' stroke='currentColor' points='27 5 72.167 50.167 27.335 95'/></svg></div>"
+          }
+        },
+        {
+          "link": {
+            "nav-item-classes": ["gallery-sidebar-nav-item"],
+            "url": "./gallery-inner.amp.html",
+            "text": "<div class='gallery-sidebar-nav-item-text'>Geo Breakfast<br/>Terra Morales</div><amp-img class='gallery-sidebar-nav-item-image' width='1' height='1' layout='responsive' src='../../img/gallery/balloon.jpg' srcset='../../img/gallery/balloon.jpg 1x, ../../img/gallery/balloon@2x.jpg 2x'></amp-img><div class='gallery-sidebar-nav-item-icon'><svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 100 100'><polyline fill='none' stroke='currentColor' points='27 5 72.167 50.167 27.335 95'/></svg></div>"
```

Comment:
could we not do the SVG as a background image on ::after?

Reply:
I'd mentioned this on Friday when we spoke, and have spent some time on it since then, but I wasn't able to get this to work.